ocs_ci/utility/rosa.py

```diff
@@ -47,11 +47,17 @@ def create_cluster(cluster_name, version):
     compute_nodes = config.ENV_DATA["worker_replicas"]
     compute_machine_type = config.ENV_DATA["worker_instance_type"]
     multi_az = "--multi-az " if config.ENV_DATA.get("multi_availability_zones") else ""
+    cluster_type = config.ENV_DATA.get("cluster_type", "")
+    provider_name = config.ENV_DATA.get("provider_name", "")
```

Comment:
Is the param `provider_name` should always exist in the provider config?

Reply:
No, this is specific for consumer deployment jobs. We need to check how multicluster deployment design to finalize it. however, for now, we are defining it in ENV_DATA using " YAML_TEXT_CONFIG"  parameter of CI.